components/BackButton/index.test.jsx

```diff
@@ -0,0 +1,26 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { useRouter } from 'next/router';
+
+import BackButton, { backButtonTestIds } from '.';
+
+describe('BackButton', () => {
+  it('renders a back button', () => {
+    render(<BackButton />);
+
+    const backButton = screen.getByTestId(backButtonTestIds.button);
+
+    expect(backButton).toBeVisible();
+  });
+
+  it('goes to the previous page upon click', () => {
```

Comment:
```suggestion
  it('routes to the previous page upon click', () => {
```

or even a step further? :) 

```suggestion
  describe('when clicked', () => {
    it('routes to the previous page', () => {
```

Reply:
done in ec0d501e73cda2a9b2acc0fbacb4ce100eaeba50